src/systems/abstractsystem.jl

```diff
@@ -1749,6 +1749,7 @@ The `simplified_sys` has undergone [`structural_simplify`](@ref) and had any occ
   - `outputs`: A vector of variables that indicate the outputs of the linearized input-output model.
   - `simplify`: Apply simplification in tearing.
   - `initialize`: If true, a check is performed to ensure that the operating point is consistent (satisfies algebraic equations). If the op is not consistent, initialization is performed.
+  - `initialization_solver_alg`: A NonlinearSolve algorithm to use for solving the initialization problem.
```

Comment:
```suggestion
  - `initialization_solver_alg`: A NonlinearSolve algorithm to use for solving for a feasible set of state and algebraic variables that satisfies the specified operating point.
```

"initialization" has no well-defined meaning when linearizing, hence the slightly different language.

Reply:
Fixed in https://github.com/SciML/ModelingToolkit.jl/pull/2762/commits/406dc9afc96b0014b1d2ed18c9475b317f20d5bc